### ISSUE TITLE
Hot reloading

### ISSUE THREAD

**Issue by lutangar** [dis-moi/extension#188]
Need to find a way to hot reload for a webextension, usually file are served by webpack-dev-server, but here it is served by the filesystem, maybe some solutions:
- https://github.com/rubenspgcavalcante/webpack-chrome-extension-reloader
> https://medium.com/front-end-hacking/hot-reloading-extensions-using-webpack-cdfa0e4d5a08

> https://stackoverflow.com/questions/2963260/how-do-i-auto-reload-a-chrome-extension-im-developing
> https://github.com/schovi/create-chrome-extension

**Comment by lutangar** [comment]
> Update: https://github.com/rubenspgcavalcante/webpack-extension-reloader

I still find this relevant and worth giving it a try!

@felix-lambert would you like to give this a try?

**Comment by JalilArfaoui** [comment]
@gregoirelacoste we talked about this …

**Comment by lutangar** [comment]
Yeah, when it's been a while, and I come back working on the extension, I realized how painful it's to work without it.
